include/zephyr/drivers/usb_c/usbc_tcpc.h

```diff
@@ -460,12 +460,11 @@ static inline int tcpc_set_roles(const struct device *dev,
  * @param dev  Runtime device structure
  * @param type  pointer to where message type is written. Can be NULL
  *
- * @retval true if message is pending, else false
+ * @retval 1 if there is pending message, else 0
```

Comment:
I think we should stick to using the Zephyr return codes. What about returning -ENODATA for no pending message, and 0 for pending message?

Reply:
Sure, I've changed existing code and this new driver